src/components/LandingAnimation/LandingAnimation.js

```diff
@@ -60,24 +62,27 @@ const SideCloudsWrapper = styled.div`
 
 const LandingAnimation = () => {
   const [isAnimationPlaying, setAnimationPlay] = useState(false);
-  const [isAnimationReplaying, setAnimationReplay] = useState(false);
+  const [isButtonInPlace, setIsButtonInPlace] = useState(false);
   const [animationButtonTextId, setAnimationButtonTextId] = useState(
     'animation.play'
   );
   const handelAnimationButtonClick = () => {
-    if (isAnimationPlaying === false) setAnimationPlay(true);
+    if (!isAnimationPlaying) {
+      setAnimationPlay(true);
+      setIsButtonInPlace(true);
+    }
   };
 
   const animationButtonRef = createRef();
-  const hiSpanRef = createRef();
-  const mySpanRef = createRef();
-  const hiMySpanRef = createRef();
-  const nameSpanRef = createRef();
-  const hiMyNameSpanRef = createRef();
-  const isSpanRef = createRef();
-  const hiMyNameIsSpanRef = createRef();
-  const karolSpanRef = createRef();
-  const hiMyNameIsKarolSpanRef = createRef();
+  const firstParagraphRef = createRef();
+  const secondParagraphRef = createRef();
+  const firstToSecondParagraphRef = createRef();
+  const thirdParagraphRef = createRef();
+  const firstToThirdParagraphRef = createRef();
+  const fourthParagraphRef = createRef();
```

Comment:
why this paragraph word here? Don't you see the problem? 
You just changed <span> to <p> and had to change also a wording here.... Let's invent some more generic name ;)